react-native/basic-example/ReactNavigationExample/README.md

```diff
@@ -16,7 +20,15 @@ inside the ios folder
 pod install
 ```
 
-## Installing/Running the App
+## Add your TalkJS App ID
+
+Update the `YOUR_APP_ID` placeholders in `App.tsx` with your TalkJS App ID. You can find your App ID in the **Settings** tab of the [TalkJS dashboard](https://talkjs.com/dashboard).
+
+## Configure push notifications
+
+Follow our React Native push notifications guide to configure push notifications for [Android](https://talkjs.com/docs/Features/Notifications/Mobile_Push_Notifications/React_Native/#android) or [iOS](https://talkjs.com/docs/Features/Notifications/Mobile_Push_Notifications/React_Native/#ios).
```

Comment:
Add this section to the React Native Navigation Example

Reply:
Done, thanks for spotting!